deployment_scripts/appimage/run_appimage.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Can you explain when this file is used and when AppRun is used?

Reply:
AppRun is part of the AppImage spec. It's the entry point. AppImage Builder generates its own AppRun from the `exec` and `exec_args` fields in `monkey_island_builder.yml`. ATM, AppRun isn't used, so I'll just remove it.